BUILD

```diff
@@ -184,6 +185,18 @@ genrule(
     heuristic_label_expansion = 0,
 )
 
+genrule(
+  name = "package-dev",
+  srcs = [ ":release" ],
+  outs = ["santa-dev.pkg"],
+  cmd = """
+  tar -xzvf $(<)
+  RELEASE_ROOT=. SCRATCH=. BUILD_DEV_DISTRIBUTION_PKG=1 \
+    ./conf/package.sh
```

Comment:
Conf

It'll only matter on a case sensitive system but just in case

Reply:
:) actually `conf` is correct. The script is being launched from the release tar.gz, which uses lowercase for its directory names.